articles/iot-edge/how-to-create-transparent-gateway-windows.md

```diff
@@ -33,35 +33,44 @@ The following steps walk you through the process of creating the certificates an
 ## Prerequisites
 1.	[Install the Azure IoT Edge runtime][lnk-install-windows-x64] on a Windows device you want to use as the transparent gateway.
 
-1. Get OpenSSL for Windows. There are many ways you can install OpenSSL. The instructions here use vcpkg to accomplish this.
-   1. Download and install vcpkg with the following commands run from an administrator PowerShell. Navigate to a directory where you want to install OpenSSL, lets call this `$VCPKGDIR`.
+1. Get OpenSSL for Windows.
 
-   ```PowerShell
-   git clone https://github.com/Microsoft/vcpkg
-   cd vcpkg
-   .\bootstrap-vcpkg.bat
-   .\vcpkg integrate install
-   .\vcpkg install openssl:x64-windows
-   ```
+>[!NOTE]
+>If you already have OpenSSL installed on your Windows device, you may skip this step but 
+> please ensure that  `openssl.exe` is available in your `%PATH%` environment variable.
+
+  There are many ways you can install OpenSSL:
+   * Download and install any [third-party OpenSSL binaries](https://wiki.openssl.org/index.php/Binaries), for example, from [this project on SourceForge](https://sourceforge.net/projects/openssl/).
+   
+   * Download the OpenSSL source code and build the binaries on your machine by yourself or do this via [vcpkg](https://github.com/Microsoft/vcpkg). 
+     * The instructions here use vcpkg to download source code, compile and install OpenSSL on your Windows machine all in very easy to use steps.
```

Comment:
Line 46 can be added to the end of line 45. (It'll look cleaner if you don't try to mix a bulleted list with a numbered list.)

Reply:
I feel it reads easier as steps if numbered. Do you suggest using further indented bullets?